src/backend/utils/mmgr/aset.c

```diff
@@ -998,6 +1010,40 @@ static void AllocSetReleaseAccountingForAllAllocatedChunks(MemoryContext context
 #endif
 }
 
+#ifdef EXTRA_DYNAMIC_MEMORY_DEBUG
+static void AllocSetPrintMemoryContextStats(MemoryContext context)
```

Comment:
It worth to have some specification and comments for interface method of AllockSetContext.